pyfr/solvers/aceuler/elements.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+from pyfr.solvers.baseadvec import BaseAdvectionElements
+
+
+class BaseFluidElements(object):
```

Comment:
Rename to `BaseACFluidElements` to avoid confusion with `BaseFluidElements` from Euler/Navier-Stokes.